conans/client/cmd/download.py

```diff
@@ -13,11 +13,12 @@ def download(ref, package_ids, remote_name, recipe, remote_manager,
     remote = registry.remotes.get(remote_name) if remote_name else registry.remotes.default
 
     hook_manager.execute("pre_download", reference=ref, remote=remote)
-    # First of all download package recipe
+
     try:
+        # First of all download package recipe
         ref = remote_manager.get_recipe(ref, remote)
     except NotFoundException:
-        raise NotFoundException("'%s' not found in remote '%s'" % (ref.full_repr(), remote.name))
+        raise RecipeNotFoundException(ref, print_rev=cache.config.revisions_enabled)
```

Comment:
Same as in the past review, shouldn't ``remote_manager.get_recipe()`` already throw a RecipeNotFoundException?

Reply:
I'd say that this won't be needed if the other issue is changed.